src/SmartComponents/Overview/Dashboard.js

```diff
@@ -30,160 +29,153 @@ import routerParams from '@redhat-cloud-services/frontend-components-utilities/f
 
 const SummaryChart = asyncComponent(() => import('../../PresentationalComponents/Charts/SummaryChart/SummaryChart'));
 const OverviewDonut = asyncComponent(() => import('../../PresentationalComponents/Charts/OverviewDonut'));
+const TagsToolbar = asyncComponent(() => import(/* webpackChunkName: "TagsToolbar" */ '../../PresentationalComponents/TagsToolbar/TagsToolbar'));
```

Comment:
Another tip here. You can use `React.lazy` and `<Suspense />` for these imports instead.

Reply:
ooo is the async component no longer supported? 🤔

Reply:
I don't know about that but its no longer necessary. It does the samé thing basically.